src/Appwrite/Platform/Modules/Proxy/Http/Rules/API/Create.php

```diff
@@ -188,6 +188,22 @@ public function action(
 
         $queueForEvents->setParam('ruleId', $rule->getId());
 
+        $certificate = $dbForPlatform->getDocument('certificates', $rule->getAttribute('certificateId', ''));
+
+        // Merge logs: priority to certificate logs if both have values, otherwise use whichever is not empty
+        $ruleLogs = $rule->getAttribute('logs', '');
+        $certificateLogs = $certificate->getAttribute('logs', '');
+        $logs = '';
+        if (!empty($certificateLogs) && !empty($ruleLogs)) {
+            $logs = $certificateLogs; // Certificate logs have priority
+        } elseif (!empty($certificateLogs)) {
+            $logs = $certificateLogs;
+        } elseif (!empty($ruleLogs)) {
+            $logs = $ruleLogs;
+        }
+        $rule->setAttribute('logs', $logs);
```

Comment:
Can be simplified to: `$logs = !empty($certificateLogs) ? $certificateLogs :  $ruleLogs` in all places